libraries/render-utils/src/Model.cpp

```diff
@@ -151,6 +151,7 @@ void Model::setOffset(const glm::vec3& offset) {
     // if someone manually sets our offset, then we are no longer snapped to center
     _snapModelToRegistrationPoint = false;
     _snappedToRegistrationPoint = false;
+    _forceOffset = true;
```

Comment:
I originally didn't see that setOffset was called (circuitously) when we're relaying an avatar's joint data to its model entity children.  instead of forcing the offset for just SkeletonModels, we can just force the offset anytime we actually set the offset, which makes more sense anyways